core/services/ardupilot_manager/mavlink_proxy/test_all.py

```diff
@@ -28,11 +45,8 @@ def test_endpoint() -> None:
 
 
 def test_endpoint_validators() -> None:
-    Endpoint.is_mavlink_endpoint({"connection_type": "tcpin", "place": "0.0.0.0", "argument": 14550})
-    Endpoint.is_mavlink_endpoint({"connection_type": "tcpout", "place": "0.0.0.0", "argument": 14550})
-    Endpoint.is_mavlink_endpoint({"connection_type": "udpin", "place": "0.0.0.0", "argument": 14550})
-    Endpoint.is_mavlink_endpoint({"connection_type": "udpout", "place": "0.0.0.0", "argument": 14550})
-    Endpoint.is_mavlink_endpoint({"connection_type": "serial", "place": "/dev/autopilot", "argument": 115200})
+    for endpoint in OUTPUT_ENDPOINTS:
+        Endpoint.is_mavlink_endpoint(endpoint.asdict())
```

Comment:
Can't we have a function in Endpoint to just call: `endpoint.is_mavlink_endpoint()`, since the function is already inside the Endpoint class it makes sense.

Reply:
We can. Will open an issue for that.

Reply:
If that's true, the test does not make sense, because you are already creating the endpoints here:
https://github.com/bluerobotics/companion-docker/pull/152/files/2970d5511111328f26168f0d9e5ea0c87bb5643c..0d63e2aea44c7c215713c0426a5e95a2b0bce0b9#diff-7498f7ef4068e0981d03dbdc5e0c34411f5fd4df2727948d43587aefac10a4cdR17
So you are testing something that was already validated by the first constructor call.

Reply:
That's true.